lib/ulid/identifier.rb

```diff
@@ -8,7 +8,7 @@ class Identifier
     include Generate
     include Compare
 
-    attr_reader :seed, :bytes, :time, :ulid
```

Comment:
👍  for me but I suppose deprecating `.ulid` could be contentious if we merge this to the public repo